### PR TITLE
X86 Backend

### PR DESCRIPTION
**Pull request by martinloretzzz**
Got the addition of two numpy arrays in x86-64 assembly running.
So all the stuff how to do the python/assembly and numpy interop is clear now, I'll work on implementing the codegen at the weekend.

### CODE REVIEW

**Comment by geohot** [comment]
A little light for a bounty lock, but given your contribution history, I locked it. Good luck!

**Comment by tinyb0t** [comment]
Changes made in `tinygrad/`:
```
------------------------------------------------------------
files                             insertions       deletions
------------------------------------------------------------
tinygrad/runtime/ops_x86.py               49               0
------------------------------------------------------------
lines added in the tinygrad folder: 49
```

File: tinygrad/runtime/ops_x86.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+from tinygrad.codegen.assembly_x86 import X86Codegen
+from tinygrad.ops import Compiled
+from tinygrad.runtime.lib import RawMallocBuffer
+from tinygrad.runtime.ops_clang import ClangProgram
+
+X86Buffer = Compiled(RawMallocBuffer, X86Codegen, ClangProgram)
```
Comment (geohot) [inline tinygrad/runtime/ops_x86.py:6]: Check out how RDNA and PTX do this, you don't need new ops.

Comment (martinloretzzz) [reply tinygrad/runtime/ops_x86.py:6]: 👍

**Pull request closed without merging**